### ISSUE TITLE
swarm -h with no LOCUST ENV vars should provide details and not a traceback

### ISSUE THREAD

**Issue by chrishare08** [SvenskaSpel/locust-swarm#1]
Running swarm -h on macOS with locust, locust-plugins, locust-swarm and granafa installed should provide detailed help as described in the project README and not a traceback because environment variables are missing.

Here is sample output

```
chare@Chriss-MBP v1 % swarm -h
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.8/bin/swarm", line 13, in <module>
    from locust_swarm.functions import *
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/locust_swarm/functions.py", line 9, in <module>
    import locust_plugins
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/locust_plugins/__init__.py", line 4, in <module>
    from .debug import run_single_user
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/locust_plugins/debug.py", line 5, in <module>
    from locust_plugins.listeners import PrintListener
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/locust_plugins/listeners.py", line 24, in <module>
    GRAFANA_URL = os.environ["LOCUST_GRAFANA_URL"]
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/os.py", line 675, in __getitem__
    raise KeyError(key) from None
KeyError: 'LOCUST_GRAFANA_URL'
```
this is a bug because setting the environment variable to a bogus value results in the help message.
```
chare@Chriss-MBP v1 % LOCUST_GRAFANA_URL="somewhere"
chare@Chriss-MBP v1 % export LOCUST_GRAFANA_URL
chare@Chriss-MBP v1 % swarm -h
usage: swarm [-h] [-f TESTPLAN] [-t RUN_TIME] [-u USERS] [--headless IGNORE_THIS] [--
...
```
## What SHOULD happen
```
chare@Chriss-MBP v1 % swarm -h
usage: swarm [-h] [-f TESTPLAN] [-t RUN_TIME] [-u USERS] [--headless IGNORE_THIS] [--
processes-per-loadgen PROCESSES_PER_LOADGEN]
             [--loadgens LOADGENS] [--loadgen-list LOADGEN_LIST] [-L LOGLEVEL] [--port PORT] [--jmeter] [--jmeter-gui]
             [--jmeter-kill-first] [--remote-master REMOTE_MASTER]

A tool for running locust/jmeter in a distributed fashion. Args that start with '--' (eg. -f) can also be set in a config file
(~/.locust.conf or locust.conf). Config file syntax allows: key=value, flag=true, stuff=[a,b,c] (for details, see syntax at
https://goo.gl/R74nmi). If an arg is specified in more than one place, then commandline values override environment variables which
override config file values which override defaults.

optional arguments:
  -h, --help            show this help message and exit
  -f TESTPLAN, --locustfile TESTPLAN
                        [env var: LOCUST_LOCUSTFILE]
  -t RUN_TIME, --run-time RUN_TIME
                        [env var: LOCUST_RUN_TIME]
  -u USERS, --users USERS
                        [env var: LOCUST_USERS]
  --headless IGNORE_THIS
                        [env var: LOCUST_HEADLESS]
  --processes-per-loadgen PROCESSES_PER_LOADGEN
                        [env var: LOCUST_PROCESSES_PER_LOADGEN]
  --loadgens LOADGENS   [env var: LOCUST_LOADGENS]
  --loadgen-list LOADGEN_LIST
                        [env var: LOCUST_LOADGEN_LIST]
  -L LOGLEVEL
  --port PORT           [env var: LOCUST_PORT]
  --jmeter              [env var: LOCUST_JMETER]
  --jmeter-gui          [env var: LOCUST_JMETER_GUI]
  --jmeter-kill-first   [env var: LOCUST_JMETER_KILL_FIRST]
  --remote-master REMOTE_MASTER
                        [env var: LOCUST_REMOTE_MASTER]

Any parameters not listed here are forwarded to locust/jmeter unmodified, so go ahead and use things like -u, -r, --host, ... All env
vars starting with LOCUST_ will be forwarded to the slaves.
```

**Comment by cyberw** [comment]
Good catch! Fixed in locust-plugins 1.0.8 (where the underlying issue was), and bumped swarm's dependency in its new release (1.0.2). So pip install -U locust-swarm should be all you need.

**Comment by cyberw** [comment]
Huh. My fix was no good. Actual fix in https://github.com/SvenskaSpel/locust-plugins/commit/8e891c4f9bfc0cafaea0c0e12ad3861991a9d716
